### ISSUE TITLE
OpenAPI spec missing some endpoints

### ISSUE THREAD

**Issue by vcampitelli** [FusionAuth/fusionauth-issues#2247]
### Description
It seems that the [OpenAPI spec](https://github.com/FusionAuth/fusionauth-openapi/blob/master/openapi.yaml) is missing some endpoints that don't have parameters in the query string.

Here are the missing endpoints:

```
DELETE /api/reactor
GET /api/connector
GET /api/consent
GET /api/entity/type
GET /api/form
GET /api/form/field
GET /api/group
GET /api/integration
GET /api/key
GET /api/messenger
GET /api/reactor
GET /api/reactor/metrics
GET /api/report/totals
GET /api/system-configuration
GET /api/system/reindex
GET /api/system/version
GET /api/tenant
GET /api/tenant/password-validation-rules
GET /api/theme
GET /.well-known/jwks.json
GET /.well-known/openid-configuration
PUT /api/entity/search
PUT /api/reactor
PUT /api/user/search
```

I've opened PR FusionAuth/fusionauth-client-builder#68 to fix the builder.

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).

**Comment by robotdan** [comment]
Does this need any doc? If not, I'll close it out.

**Comment by vcampitelli** [comment]
Hey @robotdan, sorry, this was solved on FusionAuth/fusionauth-client-builder#68 so I'm closing this.
